week-2/G-conditionals-2/exercise-4.js

```diff
@@ -9,7 +9,13 @@
 */
 
 function containsCode(sentence) {
-
+ if (sentence.indexOf("code") >= 0){
+   res = true;
```

Comment:
res is a variable without declaration. That's not good :-)
Best way though is to do this all without any variables. Remember conditional expressions will result in a boolean value already. So in this case you can directly return:
`return sentence.indexOf("code") >= 0;`